mmv1/products/compute/Autoscaler.yaml

```diff
@@ -159,7 +159,7 @@ properties:
           Defines operating mode for this policy.
         values:
           - :OFF
-          - :ONLY_UP
```

Comment:
If we change this field to a string, we can avoid a breaking change (removing `ONLY_UP` is breaking as it is deprecated) and also support any new values.